### PR TITLE
Issue #6 Updating footer and minor style changes 

### PR DESCRIPTION
**Pull request by vanshitalavange**
- Updated footer
- added products link in nav bar
- minor styling changes done

### CODE REVIEW

File: stylesheets/pages/products.css [hunk 1]
```diff
@@ -61,3 +61,9 @@
   gap: var(--spacing-sm);
   padding: 0 0 var(--spacing-xs) 0;
 }
+@media only screen and (max-width:580px){
+  .page-main{
+    grid-template-columns: 1fr;
+    gap:var(--spacing-xl);
+  }
+}
```
Comment (vaibhav-mahalle) [inline stylesheets/pages/products.css:69]: Clear and concise commit messages help the reviewer while reviewing. Keep up this good practice.

File: stylesheets/layout/layout.css [hunk 2]
```diff
@@ -159,8 +181,6 @@
   color: var(--white-color);
   background-color: var(--secondary-color);
   border-radius: var(--border-rd-md);
-  margin-top: 0.2rem;
```
Comment (vaibhav-mahalle) [inline stylesheets/layout/layout.css]: try and use px if values are too small.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some changes please look into it.

File: stylesheets/layout/layout.css [hunk 3]
```diff
@@ -52,6 +53,7 @@
   display: none;
   z-index: 2;
   box-shadow: 1px 1px 10px 3px rgba(36, 35, 35, 0.1);
```
Comment (nishasen) [inline stylesheets/layout/layout.css:55]: you can define this in variables

**Review by nishasen — APPROVED** [review]
code looks clean and properly indented, some lines could be omitted , I think its due to code wrap also you can use variables for box-shadow. rest everything looks good.

File: stylesheets/pages/products.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/pages/products.css:69]: sure

File: stylesheets/layout/layout.css (hunk 3 above)
Comment (vanshitalavange) [reply stylesheets/layout/layout.css:55]: actually the box shadows were different at all places so I decided not to use variables for that, next time onwards I will try to maintain consistency and use variables, thanks

File: stylesheets/layout/layout.css (hunk 2 above)
Comment (vanshitalavange) [reply stylesheets/layout/layout.css]: thank you for the suggestion, I'll keep this in mind next time on

**Pull request merged**